go/chat/server.go

```
@@ -2681,6 +2681,7 @@ func (h *Server) BulkAddToConv(ctx context.Context, arg chat1.BulkAddToConvArg)
 				}
 			}
 			usernamesToAdd = usernamesToRetry
+			err = nil
```

cc @joshblum I didn't want "already in channel" errors to bubble up

---

did this comeup during testing? maybe someting is broken in the retry loop then, this is supposed to retry during a race. i'd prefer squashing this on the frontend if possible, or just fixing the retry if that is casuing an issue cc @AMarcedone

---

the specific case is when everyone you're trying to add is already in the channel, so this check fails and the error doesn't have a chance to clear: `len(usernamesToAdd) > 0`. So this is just to silence that error type from ever happening.

---

can you only clear it if `len(usernamesToRetry) == 0`?

---

+1 for only clearing if `len(usernamesToRetry) = 0`

---

sure, done